### DISCUSSION TITLE
Documentation notation for AutomationCondition.cron.

### DISCUSSION THREAD

**Discussion by raiAmagi** [dagster-io/dagster#24205] [Q&A] [answered]
What can you tell us about the document ‘Declarative Automation’?

Two types are described in Automation conditions: ‘AutomationCondition.eager()’ and ‘AutomationCondition.cron(cron_schedule cron_schedule)’.

https://docs.dagster.io/concepts/automation/declarative-automation

The second, ‘AutomationCondition.cron(cron_schedule)’ appears to be incorrect as ‘cron is not described in the API documentation’.

The API documentation describes on_cron

Would it be correct to change to AutomationCondition.on_cron?

**Comment by garethbrickman** [comment, marked as answer]
Thanks for spotting that! I've put in this PR with the fix: https://github.com/dagster-io/dagster/pull/24213

**Comment by raiAmagi** [comment]
Thanks for posting the fix.
